src/kaleidoscope/hardware/technomancy/Atreus2.h

```diff
@@ -22,34 +22,38 @@
 
 #include <Arduino.h>
 #define HARDWARE_IMPLEMENTATION kaleidoscope::hardware::technomancy::Atreus2
-#include "Kaleidoscope-HIDAdaptor-KeyboardioHID.h"
 
-#include "kaleidoscope/macro_helpers.h"
+#include "kaleidoscope/hardware/technomancy/atreus2/Atreus2ScannerDescription.h"
+#include "kaleidoscope/driver/scanner/ATMegaScanner.h"
+#include "kaleidoscope/driver/led/NoLeds.h"
+#include "kaleidoscope/driver/mcu/AVR.h"
 
-#include "kaleidoscope/hardware/ATMegaKeyboard.h"
+#include "kaleidoscope/HardwareDescription.h"
+#include "kaleidoscope/Hardware.h"
 
 namespace kaleidoscope {
 namespace hardware {
 namespace technomancy {
-class Atreus2: public kaleidoscope::hardware::ATMegaKeyboard {
-  friend class ATMegaKeyboard;
+
+using Atreus2Scanner = kaleidoscope::driver::scanner::ATMegaScanner<Atreus2ScannerDescription>;
```

Comment:
Global typedef/using declaration really required?

Reply:
Nope, forgot I made it this way. Will clean up, thanks!